Amethyst Logo/README.md

```diff
@@ -0,0 +1,5 @@
+# License
+
+This logo is licensed under the [CC-BY-ND 4.0 license](https://creativecommons.org/licenses/by-nd/4.0/). You must follow the [Amethyst branding guidelines](https://www.amethyst-engine.org/branding/) when using it.
+
+Specifically, please keep in mind that you cannot modify or use the Amethyst logo in derived work without permission, and that your usage of the logo must not make it look like you are part of an official Amethyst Foundation project (when applicable).
```

Comment:
I did not choose the license, but I believe what I described was the intent we wanted. If you have an alternative license, please do (as long as this is not merged the license is not applicable).

Reply:
We might encounter trademark issues if we do not do that however.
https://www.rust-lang.org/policies/media-guide
This is how Rust does it.

Reply:
That's irrelevant. You're adding usage conditions in addition to a license that overrules any additional conditions. It will not hold up.

Reply:
If this is how Rust is doing it, Rust's conditions are as void. They either use a different/custom license, or their added clauses are as good as null.

Reply:
The logo art is CC, the brand needs to be used under special circumstances.
There is an important distinction here, otherwise one could simply make a copy of the Coca-Cola logo and license it under CC so anyone can use it.

Reply:
Rust is a trademark of the Mozilla Foundation, you probably should not worry about their situation considering they have the services of trademark lawyers.

Reply:
I looked into this CC license more carefully. While the summary says "distribute", the terms cover displaying, etc, so, non issue.

Reply:
Yes we do not limit the sharing of the work. We limit the way people display it. I think the issue is resolved.

Reply:
To back up a little bit. I have two separate concerns that I'd like to discuss a bit:

1)
I don't think the current formulation makes it clear whether we approve of the Amethyst logo being used in a certain context. Right now with this phrasing it appears like everything would require explicit approval. Even including something like a "Made using Amethyst" splash in a game. If this is the case, that would be a bit strict IMO. Other trademark policies tend to list reasonable ways to use a trademarked logo.

2)
I believe CC-BY is more suitable when coupled with a trademark policy that states that the trademark may not be reproduced in a confusing manner. A user might want to adapt the logo a bit to fit better with some specific graphic design.

Reply:
Then I think we should probably finish the trademark usage paragraph on the website that we would mirror here.